app/views/admin/cms/collections/_form.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= form_for [:admin, :cms, @resource] do |f| %>
+  <%= render "admin/cms/shared/errors" %>
```

Comment:
oh, yeah, i see why it's `@resource` now. the alternative would be to pass it in like:

`render partial: 'path/to/partial', locals: { resource: @collection }`

but not a big deal, just a suggestion